Dockerfile

```diff
@@ -4,6 +4,7 @@ MAINTAINER https://github.com/SatelliteQE
 RUN ln -s /usr/bin/python3 /usr/bin/python
 RUN dnf install -y gcc git libffi-devel openssl-devel python3-devel \
     redhat-rpm-config which libcurl-devel libxml2-devel make
```

Comment:
there was an issue with pip not available, installing python3-pip solved that for me